bin/v-create-dns-cluster-config

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# info: creates local and remote transfer keys for bind.
+# options: HOST IP
+#
+# This function creates a new transfer key for the host
+# provided and sends it to the new host
+
+#----------------------------------------------------------#
+#                Variables & Functions                     #
+#----------------------------------------------------------#
+
+# Argument definition
+host=$1
+ip=$2
+
+# Includes
+# shellcheck source=/etc/hestiacp/hestia.conf
+source /etc/hestiacp/hestia.conf
+# shellcheck source=/usr/local/hestia/func/main.sh
+source $HESTIA/func/main.sh
+# load config file
+source_conf "$HESTIA/conf/hestia.conf"
+
+#----------------------------------------------------------#
+#                    Verifications                         #
+#----------------------------------------------------------#
+
+check_args '2' "$#" 'HOST IP'
+is_system_enabled "$DNS_SYSTEM" 'DNS_SYSTEM'
+
+# Perform verification if read-only mode is enabled
+check_hestia_demo_mode
+
+#check if openssl is installed
+openssl_installed=$(which openssl | grep openssl)
+
+#----------------------------------------------------------#
+#                       Action                             #
+#----------------------------------------------------------#
+
+# Get Transfer-Key File
+if [ -e '/etc/bind/transferkeys.conf' ]; then
+	transfer_keys='/etc/bind/transferkeys.conf'
+fi
+# Get DNS Config file
+if [ -e '/etc/bind/named.conf' ]; then
```

Comment:
Check doesn't make sense it should just "select" it

Reply:
Yep.